wikiconv/conversation_reconstruction/construct_utils/conversation_constructor_test.py

```diff
@@ -0,0 +1,116 @@
+"""Tests for ReconstructConversation."""
```

Comment:
Please make sure we add the test which break the diff-match-patch previously.

Reply:
I checked the dates and the release is well after the changes.